tuist/tuist
# Fix "Embed Frameworks" build phase parameters

Resolves: https://github.com/tuist/tuist/issues/2153

### Short description 📝

Generated projects with binary Swift Package dependencies fail to build.

### Solution 📦

- Generated projects had a missing empty `dstPath` parameter for the "Embed Frameowkrs" build phase
- This small mismatch in parameters results in build failures when linking against Swift Package binary dependencies
    - I suspect Xcode doesn't recognise the generated build phase as the embed frameworks phase and ends up creating a duplicate build phase
- This was discovered while comparing projects where the same phase was added by Xcode (via the UI when embedding frameworks)

### Implementation 👩‍💻👨‍💻

- [x] Add missing build phase parameter 
- [x] Add a new fixture that depends on a binary dependency
- [x] Add acceptance test

### Test Plan 🛠

- Run `swift build && swift run tuist generate --path fixtures/ios_app_with_local_binary_swift_package`
- Verify the generated project builds
